src/AppInstallerCLITests/YamlManifest.cpp

```diff
@@ -791,8 +791,10 @@ TEST_CASE("ReadBadManifests", "[ManifestValidation]")
         { "Manifest-Bad-DuplicateKey-DifferentCase-lower.yaml", "Duplicate field found in the manifest." },
         { "Manifest-Bad-DuplicateReturnCode-ExpectedCodes.yaml", "Duplicate installer return code found." },
         { "Manifest-Bad-DuplicateReturnCode-SuccessCodes.yaml", "Duplicate installer return code found." },
+        { "Manifest-Bad-DuplicateSha256.yaml", "Multiple Installer URLs found with  to the same InstallerSha256. Please ensure the accuracy of the URLs", true },

```

Comment:
```suggestion
        { "Manifest-Bad-DuplicateSha256.yaml", "Multiple Installer URLs found with a match  to the same InstallerSha256. Please ensure the accuracy of the URLs", true },
```

is this a classic case of Trenly forgetting to commit local changes? 😁

Reply:
Yep

Reply:
Its actually a case of Trenly being an idiot